.github/workflows/python-integration-tests.yml

```diff
@@ -103,6 +103,7 @@ jobs:
 
           cd python
           poetry run pytest ./tests/integration -v
+          poetry run pytest ./tests/samples -v
```

Comment:
we also have a seperate workflow for samples, I'm wondering what will be better, this single one, or should (and can) we seperate them and have the samples be less of an issue when they don't go through?

Reply:
I think we need to place the same importance on samples (integration tests) as we do for normal integration tests. We should catch any errors right away and make sure they're fixed -- an error or breaking change, unless a service outage, should be handled ASAP.